peer/peerlist/list.go

```diff
@@ -66,6 +72,20 @@ func Capacity(capacity int) ListOption {
 	})
 }
 
+// NoShuffle disables the default behavior of shuffling peerlist order.
+var NoShuffle listOptionFunc = func(options *listOptions) {
```

Comment:
one minor issue with using `NoShuffle` instead of `NoShuffle()` is that it's a `var` that anyone can replace. For consistency with other options, and for better docs, I think it's better to keep it a function.

E.g., `NoShuffle` doesn't show up under options like the others:
https://godoc.org/go.uber.org/yarpc/peer/peerlist

Reply:
Looks like @echohead landed it as-is. :+1:

Reply:
Since it hasn't been released yet, can we update it before release?